src/portal/components/FarmNodesTable.vue

```diff
@@ -813,7 +813,10 @@ export default class FarmNodesTable extends Vue {
     }
   }
   domainCheck() {
-    return true;
+    // eslint-disable-next-line
+    const URL_REGEX = /^((?:(?:http?|ftp)[s]*:\/\/)?[a-z0-9-%\/\&=?\.]+\.[a-z]{2,4}\/?([^\s<>\#%"\,\{\}\\|\\\^\[\]`]+)?)/;
```

Comment:
I think it can accept 
- httpssss, ftpssss
- https://.rawda.com
- https://.rawda.com...., https://.rawda.com.